apps/edxapp/templates/lms/bc.yml.j2

```diff
@@ -39,8 +39,12 @@ spec:
       FROM {{ edxapp_image_name }}:{{ edxapp_image_tag }}
       USER 0
       {% if edxapp_theme_url is defined and edxapp_theme_url -%}
+      RUN apt-get update && \
+        apt-get install -y \
+        npm && \
+        rm -rf /var/lib/apt/lists/*
       COPY . /edx/app/edxapp/edx-platform/themes/custom-theme
-      RUN NO_PREREQ_INSTALL=1 paver update_assets --settings={{ edxapp_build_settings }} --skip-collect
+      RUN PATH=${PWD}/node_modules/.bin:${PATH} NO_PREREQ_INSTALL=1 paver update_assets --settings={{ edxapp_build_settings }} --skip-collect
```

Comment:
@jmaupetit  this can be removed now?

Reply:
Yes, once we've upgraded to `hawthorn.1-2.0.1`